gradle.properties

```diff
@@ -1,3 +1,3 @@
 micronautVersion=2.0.0
 libVersion=1.2.1
-testcontainersVersion=1.15.0-rc2
+testcontainersVersion=1.15.2
```

Comment:
This is to get this fix: https://github.com/testcontainers/testcontainers-java/issues/3574